src/donejs-carousel-plugin.js

```diff
@@ -345,11 +345,6 @@ export const ViewModel = DefineMap.extend({
 		this.swipeObject.startX = this.swipeObject.currentX = touchEvent.touches ? touchEvent.touches.pageX : event.clientX;
 		// dragging has started
 		this.dragging = true;
-
-		// set all slides to be visible during the transition
-		if (this.carouselOptions.transition == 'dissolve') {
```

Comment:
no longer needed because visibility is no longer a css property that i'm mutating.